test/e2e/components/vl-proza-message.js

```diff
@@ -1,7 +1,91 @@
 const { VlElement } = require('vl-ui-core');
+const { By, Key } = require('selenium-webdriver');
 
-class VlProzaMessage extends VlElement {  
+class VlProzaMessage extends VlElement {
 
+    async _getWysiwyg() {
+        return this.findElement(By.css('#wysiwyg'));
+    }
+
+    async _waitUntilEditable() {
+        return this.driver.wait(async () => {
+            return this.isEditable();
+        }, 5000);
+    }
+
+    async getText() {
+        const wysiwyg = await this._getWysiwyg();
+        return this.driver.executeScript('return arguments[0].innerText', wysiwyg);
+    }
+
+    async shiftEnter() {
+        const wysiwyg = await this._getWysiwyg();
+        const shiftEnter = Key.chord(Key.SHIFT, Key.ENTER);
+        return wysiwyg.sendKeys(shiftEnter);
+    }
+
+    async clear() {
+        const wysiwyg = await this._getWysiwyg();
+        return this.driver.executeScript('return arguments[0].innerText = ""', wysiwyg);
+    }
+
+    async exitEditMode() {
+        const wysiwyg = await this._getWysiwyg();
+        return wysiwyg.sendKeys(Key.ESCAPE);
+    }
+
+    async confirm() {
+        const wysiwyg = await this._getWysiwyg();
+        return wysiwyg.sendKeys(Key.ENTER);
+    }
+
+    async clickOnPencil() {
```

Comment:
enterEditMode zou ik eerder zeggen.
clickOnPencil is een detail, niet de API.

Reply:
Correct, aangepast.